### ISSUE TITLE
[Chat - Compose region]: Screen reader is announcing code information along with message in "Type a new message" edit while renavigating. 

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13390]
Test Environment:
Platform: Web-chrome
chrome Version 93.0.4577.63 (Official Build) (64-bit)
OS Build: Dev (OS Build 22130.1000) 
Ring Version: Microsoft teams R1
Tool: Accessibility insights for web
Screen reader: NVDA 2021.1

Prerequisite:
Login with below credentials.
1.Username: v-
2.External: Hcltester1@outlook.com.

Repro-Steps:
1.Launch teams.live.com
2.Login with external credential's
3.Now navigate to new chat button -> Enter v- -> Start a new message 
4.Type hello 
5.Renavigate to type a new message edit field
<img width="961" alt="MAS1 3 1_Screen reader is announcing code information along with message in Type a new message edit while renavigating" src="https://user-images.githubusercontent.com/93735775/155614070-0980fe20-5968-40df-9499-b03c8c18db2e.png">

6.Observe the screen reader announcement whether it is announcing incorrect information or not.

Actual result:
Screen reader is announcing code information along with message in type a new message edit while renavigating. Here it is announcing like <p></p> edit multiline text hello.

Expected result:
Screen reader should announce only the message information while renavigating i.e. edit multiline text hello.

User Impact:
Screen reader users will get impacted if the Screen reader is announcing code information along with message in type a new message edit while renavigating.
<img width="961" alt="MAS1 3 1_Screen reader is announcing code information along with message in Type a new message edit while renavigating" src="https://user-images.githubusercontent.com/93735775/155613825-62c85015-7a83-4855-8271-f7bdabfd40af.png">

**Comment by seanbudd** [comment]
The actual and expected steps appear to be the same: "edit multiline text hello".
These needs a **minimal** HTML example and clearer steps to reproduce. 
Please create a [codepen](https://codepen.io) and fill out the issue template.
Closing this issue, please open a new issue with these changes.
